packages/fx-core/templates/bicep/identity.provision.orchestration.bicep

```diff
@@ -0,0 +1,14 @@
+// Resources for identity
+module userAssignedIdentityProvision './provision/identity.bicep' = {
+  name: 'userAssignedIdentityProvision'
+  params: {
+    provisionParameters: provisionParameters
+  }
+}
+
+output identityOutput object = {
+  teamsFxPluginId: 'fx-resource-identity'
```

Comment:
This name should be consistent to ComponentNames.Identity

Reply:
It is fixed.